libraries/AP_RCProtocol/AP_RCProtocol_SBUS.cpp

```diff
@@ -205,8 +205,8 @@ bool AP_RCProtocol_SBUS::sbus_decode(const uint8_t frame[25], uint16_t *values,
 void AP_RCProtocol_SBUS::process_pulse(uint32_t width_s0, uint32_t width_s1)
 {
     // convert to bit widths, allowing for up to 1usec error, assuming 100000 bps
```

Comment:
Should this comment now be `up to 4us error`?

Reply:
@WickedShell Yes, you are right. Fixed comment here and in AP_RCProtocol_DSM.cpp which also had not actual comment.